contrib/platform/src/com/sun/jna/platform/win32/WinUser.java

```diff
@@ -2088,4 +2088,10 @@ public String toString() {
      * Bitmask for the RESERVED2 key modifier.
      */
     int MODIFIER_RESERVED2_MASK = 32;
+
+    /**
+     * Indicate that information is needed about the object itself.
+     */
+    int CHILDID_SELF = 0;
```

Comment:
I was using this as a magic number so going to add it in as a proper constant instead.
Someone please double check for me that this constant appears in `WinUser.h`